Makefile

```diff
@@ -6,3 +6,9 @@ site:
 
 build:
 	gatsby build && gatsby serve
+
+outdated:
```

Comment:
What's the purpose of adding these commands?
We don't want to update the dependencies always, because a lot times new releases can break a lot of things.

Reply:
Okay, I thought having this would be a good option 😶
But will change it back

Reply:
Doesn't make much sense, to me.

Reply:
Yes I completely agree, I also thought of it again. Have removed those commands from the makefile and updated the PR